PatientRegistrationApp/src/components/HouseholdReviewSubmit.vue

```diff
@@ -1,22 +1,162 @@
 <template>
 	<v-container fluid>
-	
-		<v-row
-			align="center"
-			justify="center"
-		>
-		<p>Placeholder for HouseholdReviewSubmit</p>
+		<v-row align="center" justify="center">
+			<v-heading> Please review all entries before submitting the registration! </v-heading>
+		</v-row>
+		<v-row>
+			<v-col cols="12" sm="6" md="3">
+				<div class="font-weight-medium primary--text">Household Home Address </div>
+			</v-col>
+		</v-row>
+
+		<v-row>	
+			<v-col cols="12">
+				<div class="font-weight-medium">Street Address:  <span class="font-weight-regular">{{dataHouseholdHomeAddress.householdLineAddress}}</span></div>
+				<div class="font-weight-medium">City, County, State, Country, Zip Code:  <span class="font-weight-regular">{{dataHouseholdHomeAddress.householdCityAddress}}, 
+					{{dataHouseholdHomeAddress.householdDistrictAddress}}, {{dataHouseholdHomeAddress.householdStateAddress}}, {{dataHouseholdHomeAddress.householdCountryAddress}}, {{dataHouseholdHomeAddress.householdPostalCode}}</span></div>
+			</v-col>
+		</v-row>
+
+		<v-row>
+			<v-col cols="12" sm="6" md="3">
+				<div class="font-weight-medium primary--text">Household Contact Information</div>
+			</v-col>
+		</v-row>
+
+		<v-row>	
+			<v-col cols="12">
+				<div class="font-weight-medium">Primary Phone: <span class="font-weight-regular">{{dataHouseholdContactInfo.primaryPhoneNumber}} 
+					({{dataHouseholdContactInfo.primaryPhoneNumberType}})</span></div>
+				<div class="font-weight-medium">Secondary Phone: <span class="font-weight-regular">{{dataHouseholdContactInfo.secondaryPhoneNumber}} 
+					({{dataHouseholdContactInfo.secondaryPhoneNumberType}})</span></div>
+				<div class="font-weight-medium">Primary E-mail: <span class="font-weight-regular">{{dataHouseholdContactInfo.primaryEmail}}</span></div>
+				<div class="font-weight-medium">Secondary E-mail: <span class="font-weight-regular">{{dataHouseholdContactInfo.secondaryEmail}}</span></div>
+				<div class="font-weight-medium">Follow-up approval: <span class="font-weight-regular">{{dataHouseholdContactInfo.approval}}</span></div>
+			</v-col>
+		</v-row>
+
+		<v-row>	
+			<v-col cols="12">
+				<v-card class="d-flex flex-wrap" flat>
+					<v-card class="pa-2" flat min-width=33%>
+						<div class="font-weight-medium primary--text">Personal Info: Household Member #1</div>
+						<div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].householdFamilyName}}, 
```

Comment:
Is the `...[0]...` basically why your code only shows the first patient? If so, are you trying to set up a framework to accommodate the future card or is all of that work also handled in this new card?

Reply:
Some type of loop might be good for it.?

Reply:
I do patient 1 separately (because they have a different emergency contact), but I do a for loop for patients 2-20 (starting at line 56).  MVS-385 will take care of populating the data for patients 2-20.